File: JobAggregator/JobAggregator.Core/Entities/Organization.cs
using System.ComponentModel.DataAnnotations;

namespace JobAggregator.Core.Entities
{
    public class Organization : Entity
    {
        [Required]
        public string Name { get; set; }
        public string? Website { get; set; }
        [EmailAddress]
        [Required]
        public string Email { get; set; }
        public int? PhoneNumber { get; set; }
        public string? Address { get; set; }
        [Required]
        public List<User> Users { get; set; }
        public List<Activity>? Activities { get; set; }
        public List<Vacancy>? Vacancies { get; set; }
        public List<OrganizationAplication>? OrganizationAplications { get; set; }
    }
}

File: JobAggregatorFront/src/app/models/organization.ts
import { Handbook } from "./handbook";
import { User } from "./user";
import { Vacancy } from "./vacancy";

export interface Organization{
    id: number;
    name: string;
    website?: string;
    email: string;
    phoneNumber?: number;
    address?: string;
    activities?: Handbook[];
    users: User[];
    vacancies?: Vacancy[];
}